templates/dashboard.html

```diff
@@ -1,230 +1,87 @@
-
-            <!-- Card 5 -->
-            <div class="border-2 border-deep-blue rounded-lg p-6 flex flex-col items-center justify-center shadow-md text-center">
-                <form class="w-full">
-                    <!-- Kelas Input Field -->
-                    <div class="mb-4 w-full">
-                        <input type="text" id="kelas1" class="bg-gray-50 border border-gray-300 shadow-md shadow-gray-500 text-gray-900 text-sm rounded-lg w-40 p-2.5 mb-10 mx-auto block text-center" placeholder="Kelompok" required />
-                    </div>
-
-                    <!-- Image Display -->
-                    <img src="/static/images/File.png" alt="Image 1" class="h-14 mb-4 mx-auto">
-
-                    <!-- File Upload Section -->
-                    <label for="file-upload-5" class="text-teal-600 font-semibold cursor-pointer">
-                        <input type="file" id="file-upload-5" class="hidden" onchange="displayFileName()" />
-                        Upload File
-                    </label>
-                    
-                    <!-- Display File Name -->
-                    <div id="file-name-5" class="mt-2 text-gray-500"></div>
-                </form>
-            </div>
-
-           <!-- Card 6 -->
-            <div class="border-2 border-deep-blue rounded-lg p-6 flex flex-col items-center justify-center shadow-md text-center">
-                <form class="w-full">
-                    <!-- Kelas Input Field -->
-                    <div class="mb-4 w-full">
-                        <input type="text" id="kelas2" class="bg-gray-50 border border-gray-300 shadow-md shadow-gray-500 text-gray-900 text-sm rounded-lg w-40 p-2.5 mb-10 mx-auto block text-center" placeholder="Kelompok" required />
-                    </div>
-
-                    <!-- Image Display -->
-                    <img src="/static/images/File.png" alt="Image 1" class="h-14 mb-4 mx-auto">
-
-                    <!-- File Upload Section -->
-                    <label for="file-upload-6" class="text-teal-600 font-semibold cursor-pointer">
-                        <input type="file" id="file-upload-6" class="hidden" onchange="displayFileName()" />
-                        Upload File
-                    </label>
-                    
-                    <!-- Display File Name -->
-                    <div id="file-name-6" class="mt-2 text-gray-500"></div>
-                </form>
+    <section class="max-w-6xl mx-auto p-8">
+        {% if files %}
+        <!-- File Cards Container -->
+        <div id="files-container" class="grid grid-cols-1 md:grid-cols-4 gap-6 mb-8">
+            {% for file in files %}
+            <div
+                class="border-2 border-deep-blue rounded-lg p-6 flex flex-col items-center justify-center shadow-md text-center">
+                <img src="/static/images/File.png" alt="File Icon" class="h-14 mb-4" />
+                <div class="text-deep-blue font-bold text-center">{{ file[-30:].strip() }}</div>
```

Comment:
let me guess, the reason why you do this `file[-30:].strip()` is because you want to remove the directory name is that right?

if it is then please use a better way to get the filename, for example, by splitting the text with `"/"` and then get the last element in the array

Reply:
Noted on that one, sir!

Reply:
there are some files names that are sooooo long but I guess I'll just choose to show the whole name